test/alternative-index-method.test.ts

```diff
@@ -5,7 +5,7 @@ import { Tweet } from './models/tweet'
 describe('Index Method', function () {
 
   beforeAll(async function () {
-    await mongoose.connect(config.mongoUrl, config.mongoOpts)
+    await mongoose.connect(config.mongoUrl)
```

Comment:
`mongoose.connect()` doesn't accept options anymore?

Reply:
Yes, `mongoose.connect()` does not accept the passed options anymore.